project/tests_codeception/Acceptance/Test02_LoginCest_homeCest.php

```diff
@@ -11,7 +11,7 @@ public function login(AcceptanceTester $I)
     {
         $I->wantTo('Login new user');
 
-        $I->amOnPage("/home");
```

Comment:
Wydaje mi się, że dobrze byłoby, aby strona /home przekierowywała na stronę główną + było chronione logowaniem. Pewnie należałoby ten pomysł poddać na daily @serwanska . Jeśli miałaby tu być pusta linia to można usunąć ten test, bo wtedy jest taki sam jak Test01.